@MartinNowak what does this error mean?
```
timelimit -t 90  generated/linux/release/64/unittest/test_runner std.xml
0.005s PASS release64 std.xml
posix.mak:379: recipe for target 'unittest/std/xml.run' failed
```

--- a/src/ddmd/backend/cgcod.c
+++ b/src/ddmd/backend/cgcod.c
@@ -149,9 +149,9 @@ void codgen(Symbol *sfunc)
     Alloca.init();
     anyiasm = 0;
 
-    if (config.exe & (EX_LINUX | EX_OSX | EX_OSX64 | EX_FREEBSD | EX_OPENBSD | EX_SOLARIS))
+    if (config.ehmethod == EH_DWARF)
     {
-        /* The dwarf unwinder for some platforms relies on the function epilog to exist
+        /* The dwarf unwinder relies on the function epilog to exist
          */
         for (block* b = startblock; b; b = b->Bnext)
         {

--- a/src/ddmd/s2ir.d
+++ b/src/ddmd/s2ir.d
@@ -958,6 +958,7 @@ private extern (C++) class S2irVisitor : Visitor
         const int rtlthrow = config.ehmethod == EHmethod.EH_DWARF ? RTLSYM_THROWDWARF : RTLSYM_THROWC;
         e = el_bin(OPcall, TYvoid, el_var(getRtlsym(rtlthrow)),e);
         block_appendexp(blx.curblock, e);
+        block_next(blx, BCexit, null);          // throw never returns
     }
 
     /***************************************

